src/dialects/mssql/query-generator.js

```diff
@@ -397,16 +397,15 @@ export class MsSqlQueryGenerator extends AbstractQueryGenerator {
 
     const commands = [];
     let offset = 0;
-    const batch = Math.floor(250 / (allAttributes.length + 1)) + 1;
     while (offset < Math.max(tuples.length, 1)) {
-      const tupleStr = tuples.slice(offset, Math.min(tuples.length, offset + batch));
+      const tupleStr = tuples.slice(offset, Math.min(tuples.length, offset + 1000));
```

Comment:
Can you add the following comment here?

```suggestion
      // SQL Server can insert a maximum of 1000 rows at a time,
      // This splits the insert in multiple statements to respect that limit
      const tupleStr = tuples.slice(offset, Math.min(tuples.length, offset + 1000));
```